fedimint-cli/src/client.rs

```diff
@@ -59,6 +59,8 @@ pub enum ClientCmd {
     Info,
     /// Reissue notes received from a third party to avoid double spends
     Reissue { oob_notes: OOBNotes },
+    /// Parse notes from a string and return them as JSON
+    ParseJson { oob_notes: OOBNotes },
```

Comment:
I'd definitely rename it, `ParseJson` is just way to generic. I could imagine a command under `dev`, i.e. `fedimint-cli dev <name>`. Where `<name>` is either `decode-ecash` or we make `decode` its own subcommand: `fedimint-cli dev decode ecash <notes>` and e.g. extend it later on to also contain the invite decoding `fedimint-cli dev decode invite <invite_code>`.